flake/modules/nixos/option.nix

```diff
@@ -0,0 +1,19 @@
+{
+  pkgs,
+  lib,
+  ...
+}:
+let
+  inherit (lib.options) mkOption;
+  inherit (lib.types) str;
+in
+{
+  options.programs.schizofox = {
+    userName = mkOption {
+      type = str;
+      default = "";
+      description = "Name of the user to install the configuration for";
+      example = "ryan";
+    };
+  };
+}
```

Comment:
Schizofox has no place asking for a username. Please refer to my previous comment.

```suggestion
```